x-pack/plugins/observability_solution/dataset_quality/server/routes/data_streams/get_degraded_field_analysis/index.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ElasticsearchClient } from '@kbn/core-elasticsearch-server';
+import { DegradedFieldAnalysis } from '../../../../common/api_types';
+import { createDatasetQualityESClient } from '../../../utils';
+import { getDataStreamMapping } from './get_datastream_mappings';
+import { getDataStreamSettings } from './get_datastream_settings';
+
+// TODO: The API should also in future return some analysis around the ignore_above check.
+// As this check is expensive and steps are not very concrete, its not done for the initial iteration
```

Comment:
Do you have any issue to reference here?

Reply:
Yes, the original project links to an issue where you will see this comment from Felix.
https://github.com/elastic/observability-dev/issues/3499#issuecomment-2246034361

Here you will see the heuristics.

BTW, this should be `ignore_malformed` and not `ignore_above`. I will fix it